apache-centos7-atomicapp/Nulecule

```diff
@@ -6,6 +6,10 @@ metadata:
   name: Apache CentOS Atomic App
   appversion: 0.0.1
   description: Atomic app for deploying a really basic Apache HTTP server on CentOS
+params:
+    - name: provider
+      description: The specified default provider.
+      default: docker
```

Comment:
This is purposeful, just because user can know that he/she can change provider here.